pubtools/pulplib/_impl/schema/repository.yaml

```
@@ -83,6 +83,15 @@ properties:
       pub_temp_repo:
         type: boolean
 
+      # True if repository supposed to be populated from repositories
+      # mentioned in pupulation_sources
+      ubi_population:
+        type: boolean
+
+      # List of repositories used for populating the repository
+      population_sources:
+        type: array
```

Can we please validate the contents of the array as well, it's easy:

```suggestion
        type: array
        items:
          type: string
```